content/applications/marketing/events/revenues_report.rst

```diff
@@ -0,0 +1,196 @@
+- :guilabel:`Registration Status`: group data based on the status of registrations.
+- :guilabel:`Sale Order Status`: group data based on the status of event-related sales orders.
+- :guilabel:`Customer`: group data based on customer records.
+- :guilabel:`Add Custom Group`: click the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon to
+  reveal a drop-down of grouping options to choose from. To select one, click on the desired option
+  from the drop-down menu, and Odoo adds it to the :guilabel:`Group By` column. Multiple selections
+  can be made.
+
+Sample report: event ticket analysis (graph)
+============================================
+
+The following is an example of how various filters and grouping options can be utilized to create a
+useful analytic graph report related to event revenues. In this case, the configurations present
+data about sold or free tickets to published events, with the metrics separated by ticket type and
+event.
+
+.. image:: revenues_report/event-ticket-analysis.png
+   :align: center
+   :alt: The event ticket analysis sample report with unique filters and groupings in place.
+
+To create such a report, navigate to :menuselection:`Events app --> Reporting --> Revenues'. Stay in
+the default graph view, but remove the default filters from the :guilabel:`Search...` bar.
+
+Then, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` to the right of the
+:guilabel:`Search...` bar, to reveal the drop-down mega menu of filter and grouping options.
+
+From here, select the :guilabel:`Free` and :guilabel:`Sold` from the :guilabel:`Filters` column.
+
+Then, since it is desired to **only** view data related to already published events, select the
+:guilabel:`Published Events` option in the :guilabel:`Filters` column, as well.
+
+Next, in the :guilabel:`Group By` column, select the :guilabel:`Event` and :guilabel:`Ticket`
+options, in that sequential order. Doing so first groups the data by event *then* by ticket type,
+and not the other way around, which provides a more useful array of data to analyze.
+
+.. important::
+  The order in which the options are selected in the :guilabel:`Group By` column directly affects
+  how the data is presented (and visualized) on the report.
+
+From there, additional configurations can be added for more detailed data, if desired.
+
+If no additional filters or groupings are added, Odoo presents a graphical representation of data
+related to all *free* or *sold* tickets for *published events*, grouped by *event*, and organized by
+*ticket* type.
+
+Sample report: event type analysis (pivot table)
+================================================
+
+The following is an example of how various filters and grouping options can be utilized to create a
+useful analytic pivot table report related to event revenues. In this case, the configurations
+present data about how much revenue different event types have generated, in order to gauge which
+events are the most profitable.
```

Comment:
Do you want to perhaps add that in the pivot, they can export the data and add to dashboards or reports within Odoo - and that could be a value add, or why someone would want to showcase the data in a pivot rather than a graph?

Reply:
I fully understand (and appreciate) the detailed suggestion. I completely get where you're coming from. However, I feel like that sort of addition may "muddy the waters" a bit. At this point in the doc, I've provided the user/reader with adequate information to do as they please on this reporting page, and I'm not entirely sure another lengthy tangent of this nature would be super useful, as they may already be aware of such functionality from an essentials doc. But, again, I really appreciate the suggestion and attention to detail!